src/Halogen/Driver.purs

```diff
@@ -131,7 +131,7 @@ runUI c s element = _.driver <$> do
         when (isJust rp) $ render ref
         putVar rpRef Nothing
         q
-      HaltHF -> empty
+      HaltHF msg -> throwError $ error msg
```

Comment:
OK, then I guess I'm wrong :). I only brought it up because I know whenever we've had unhandled exeption in the FFI crop up during rendering it totally nukes the driver (app becomes unresponsive to inputs, no errors show in the console). But if it's that case that `empty` throws an error, are these errors even propagated anywhere? I've can't say I've ever seen the "Always fails" error, and I've definitely triggered Halt conditions. Maybe its something worth testing?